source/common/upstream/upstream_impl.cc

```diff
@@ -39,6 +39,8 @@
 
 #include "extensions/transport_sockets/well_known_names.h"
 
+#include "spdlog/spdlog.h"
```

Comment:
no spdlog outside of logger library, I don't think this is directly used?

Reply:
right, leftover of debugging..

Reply:
resolving.